Documentation/devel/stage1-implementors-guide.md

```diff
@@ -118,3 +118,22 @@ Examples
     ]
 }
 ```
+
+Filesystem Layout Assumptions
+-----------------------------
+
+### opt/stage2
+
+As mentioned above, in rkt, the apps will be executed during stage2.
+rkt expects the directory path `opt/stage2` to exist under the stage1 rootfs.
```

Comment:
@iaguis: So it looks like I can drop the creation of the `/opt/stage2` directory tree from the build system, right?